apps/web/pages/api/book/event.ts

```diff
@@ -507,7 +583,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   let booking: Booking | null = null;
   try {
     booking = await createBooking();
-    evt.uid = booking.uid;
+    evt.uid = booking ? booking.uid : null;
```

Comment:
You can refactor this by one of these approaches:
- Optional chaining
  ```js
  evt.uid = booking?.uid;
  ```
  In this case, `evt.uid` will be `undefined` when `booking` is null.

- Null coalescing operator
  ```js
  evt.uid = booking?.uid ?? null;
  ```
  In this case, `evt.uid` will be `null` when `booking` is null.

By using optional chaining, the code will be cleaner but I am not sure if `evt.uid` can be `undefined`.

What do you think?

Reply:
Hi Miguel, yes the `_evt.uid_` could be `undefined`.
![image](https://user-images.githubusercontent.com/19562383/162845394-3b3842db-af17-4086-8cdb-e3e0e365d84e.png)

Reply:
Hmm, but the definition type inside the interface `CalendarEvent` specifies that it should be a string or null... So IMHO let's keep it null when there is no booking.